README.md

```diff
@@ -6,11 +6,155 @@
 
 ## Usage
 
-Coming soon!
+### TypeScript (NPM)
 
-## Contributing/Security
+> Repository: https://www.npmjs.com/package/cdk-monitoring-constructs
 
-See [CONTRIBUTING](CONTRIBUTING.md) for more information.
+Add the dependency to your `package.json`:
+
+```json
+{
+  "dependencies": {
+    "cdk-monitoring-constructs": "^0.0.11",
+
+    // peer dependencies
+    "constructs": "^3.3.69",  // 3.3.69 >= x < 4.0.0
+    "monocdk": "^1.123.0",    // 1.123.0 >= x < 2.0.0
```

Comment:
We probably don't need to explain the semver ranges, npm users should normally be familiar with it.